src/client/components/PaginationFooter.tsx

```diff
@@ -37,12 +40,11 @@ const PaginationFooter = ({ currentPage, totalPages, setPage }: StateProps) => {
     pageNumbers &&
     pageNumbers.map(number => {
       return (
-        <li
-          key={number}
-          id={number.toString()}
-          sx={number === currentPage ? style.paginationSelected : style.pagination}
-        >
-          <Button disabled={number === currentPage} onClick={() => setPage(number)}>
+        <li sx={{ display: "inline" }} key={number} id={number.toString()}>
+          <Button
+            sx={number === currentPage ? style.paginationSelected : style.pagination}
+            onClick={() => setPage(number)}
```

Comment:
Since we're not disabling the button, we need to update the `onClick` to not make an AJAX call when you click on the button for the current page:
```suggestion
            onClick={() =>number !== currentPage && setPage(number)}
```